Makefile

```diff
@@ -4,7 +4,8 @@ UI_IMAGE = quay.io/ztpfw/ui
 BRANCH ?= $(shell git branch --show-current | tr '[:upper:]' '[:lower:]' | tr '\/' '-')
 HASH := $(shell git rev-parse HEAD)
 RELEASE ?= latest
-EDGECLUSTERS_FILE ?= ${PWD}/hack/deploy-hub-local/edgeclusters.yaml
+CLUSTER_NAME ?= edgecluster
```

Comment:
this will be the default value...If you want to pass as var using CLUSTER_NAME=xxx make yyy it will not work...If you want to pass the var, you don't need to set it and just get the variable with $(CLUSTER_NAME)...
It's something which we discovered few days ago

Reply:
mmmhhh.... mmmmhhhhh.... 

![image](https://user-images.githubusercontent.com/13816/181712402-993cf00e-b24f-4cf4-9ecc-8eb1db540339.png)


![image](https://user-images.githubusercontent.com/13816/181712436-52eb3a89-1443-4f1e-9e78-72575d1cf338.png)

Reply:
good to know ;) resolved